src/frame-writer.cpp

```diff
@@ -355,6 +372,32 @@ FrameWriter::FrameWriter(const FrameWriterParams& _params) :
     }
 }
 
+void FrameWriter::convert_pixels_to_yuv(const uint8_t *pixels,
+    const uint8_t *formatted_pixels, int stride[])
+{
+    bool y_invert = (pixels != formatted_pixels);
+    bool converted_with_opencl = false;
+
+#ifdef HAVE_OPENCL
+    if (params.opencl && params.force_yuv)
+    {
+        int r = opencl->do_frame(pixels, encoder_frame,
+            get_input_format(), y_invert);
+
+        converted_with_opencl = (r == 0);
+    }
+#else
+    /* Silence compiler warning when opencl is disabled */
+    (void)(y_invert);
```

Comment:
we can move this inside the #ifdef

Reply:
The point of this is to not leave y_invert variable declared but not used in the case where opencl is disabled. If we put it inside the ifdef, the warning still happens.